More miscellaneous runtime parameter changes for beta05

gustavo-marques opened the pull request:
- Adds new topographic edits and updated channel list.  These lead to slightly improved circulation and hydrograph (https://github.com/NCAR/omwg_dev/discussions/55)
- Enabled COMPUTE_MONIN_OBUKHOV = True to improve the OBL seasonal cycle by limiting the OBL depth to no deeper than the Monin-Obukhov depth.  Because of this change, we now need to assure that OBL is always >= than the first thickness (dz) used in the model. This is done by setting `MINIMUM_OBL_DEPTH = 2.5`.
- Updated background mixing parameters:  
  - KV = 1.0E-06  
  - KD = 1.0E-07  
  - PRANDTL_BKGND = 10  
- Enable bug fixes introduced in https://github.com/mom-ocean/MOM6/pull/1647 

Fixes: https://github.com/ESCOMP/MOM_interface/issues/218, https://github.com/ESCOMP/MOM_interface/issues/213, https://github.com/ESCOMP/MOM_interface/issues/208, https://github.com/ESCOMP/MOM_interface/issues/220

gustavo-marques commented:
I will add the new files to input_data_list.yaml later today.

gustavo-marques commented:
I've added MINIMUM_OBL_DEPTH = 2.5 for cases using tx2_3v2. This solves the issue with the GNU debug test reported in https://github.com/NCAR/MOM6/pull/331

alperaltuntas commented:
Thanks, @gustavo-marques. I'll re-run the test suite.

alperaltuntas reviewed (changes requested):
Dimensional scaling test for the L dimension fails when this PR gets merged. I am investigating.

alperaltuntas commented:
@gustavo-marques Is this ready for testing now?

gustavo-marques commented:
Yes, thanks!

gustavo-marques commented:
With the latest beta05 update, we are getting negative Laplacian viscosities near coastlines. @iangrooms  thinks this is a bug; fixing it will take a few days (perhaps more). To avoid additional delays in tag creation, we suggest going back to `LEITH_AH`.

I am also reverting to using a constant frontal length scale for MLE. As Gokhan pointed out yesterday, the MLE overturning with the variable frontal scale is too strong, reverting the near-surface residual MOC in the Southern Ocean (see plot below).
For now, including this in the tag does not make sense. 


![image](https://github.com/user-attachments/assets/39941b51-9316-4f92-b9c3-853787623f48)

gustavo-marques commented:
I updated the initial description by removing the following:

- Configured MLE settings (this relies on PR [#331](https://github.com/NCAR/MOM6/pull/331)):  
  - Set MLE_FRONT_LENGTH_FROM_FILE = True  
  - Updated MLE_FL_FILE to "mle-lf-clim-tx2_3v2_20250115.nc"  
- Enabled Leithy parameterization with:  
  - USE_LEITHY = True  
  - LEITH_BI_CONST = 90.0  
  - LEITHY_CK = 1.0 (full backscatter)  
  - LEITH_AH = False
